frontend/src/components/Pages/UnauthorizedPage/UnauthorizedPage.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { GoToAccessITButton } from './GoToAccessITButton'
+
+const StyledUnauthorizedPage = styled.div`
```

Comment:
This styling is duplicated in two files, maybe refactor the page component to be reusable for both